cmd/nitrite/main.go

```diff
@@ -40,7 +41,15 @@ func main() {
 		os.Exit(2)
 	}
 
-	verifier, err := nitrite.New(nitrite.WithAllowDebug(*fAllowDebug))
+	var certProvider = nitrite.EmbeddedNitroCertProvider
+	if *fAllowSelfSigned {
+		certProvider = nitrite.SelfSignedCertProvider
+	}
+
+	verifier, err := nitrite.New(
+		nitrite.WithAllowDebug(*fAllowDebug),
+		nitrite.WithCertProvider(certProvider),
+	)
```

Comment:
Added support for verifying self-signed attestations, so that we can use that in https://github.com/blocky/sequencer/blob/main/verify.sh if necessary

Reply:
Could you add a todo here with a link to the card for removing self-signed cert code similar to what we do in document:
```golang
	// TODO: remove the support for self-signed attestations as part of
	//  https://blocky.atlassian.net/browse/BKY-5620 (remove !cert.IsCA path)
```

Reply:
done